lib/sprockets/manifest.rb

```diff
@@ -248,11 +248,12 @@ def clean(count = 2, age = 3600)
       asset_versions.each do |logical_path, versions|
         current = assets[logical_path]
 
+        i = 0
         versions.reject { |path, _|
           path == current
         }.sort_by { |_, attrs|
           # Sort by timestamp
-          Time.parse(attrs['mtime'])
+          [Time.parse(attrs['mtime']), i += 1]
```

Comment:
I don't think we need stable sort here. it is very unlike that we will have two asset version with the same mtime.

Reply:
this test fails without it (ruby 2.4 and ruby 2.5)
```
TestManifest#test_"remove old backups(count)" [C:/projects/sprockets/test/test_manifest.rb:452]:
Expected false to be truthy
```
https://ci.appveyor.com/project/rafaelfranca/sprockets/build/1.0.606/job/f9442f2r8kc4mfv6

Reply:
I think it is mostly the test is trying to exercise a case that doesn't happen in real world. I'd prefer to change the test to generate mtimes that are not the same.

Reply:
That test doesn't test what it looks like. The time is not based on actual mtime of the file, but rather on the current time at `manifest.compile`

```
        mtime = Time.now.iso8601
```